### PR TITLE
Update rate limits dashboard screenshot to show trendline

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Update rate limits dashboard screenshot to show trendline in a rolling window
- **Is this PR related to a Monolith release?** Yes 2021.08.0

### Resolves:

* [OKTA-415928](https://oktainc.atlassian.net/browse/OKTA-415928)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ When a rate limit violation occurs, you're notified in the following ways and ca
 
 * You receive a rate limit email that informs you that you've hit a rate limit for an endpoint. Click **View utilization history** in the email to navigate to the rate limits dashboard and System Log for that particular endpoint.
 
-![Access the dashboard by email notification](/img/RLDashboard1.png "Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard")
+![Access the dashboard by email notification](/img/RLDashboard1.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:43]:
```suggestion
![Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard](/img/RLDashboard1.png "Access the dashboard by email notification")
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:43]: I'm not sure if you wanted to keep the image title or keep the alt-text? Based on the properties of the rendered image, the first string between [] is the alt-text, the second string in quotes is the image title.

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ When a rate limit violation occurs, you're notified in the following ways and ca
...
 
 * You've hit a rate limit when you're in the Admin Console, and a banner appears to inform you. Click **utilization history** in the banner to open the rate limits dashboard for that particular endpoint or **syslog** to open the System Log.
 
-![Access the dashboard by link in the banner](/img/RLDashboard2.png "Displays the banner in the dashboard that notifies you of a rate limit violation with the link to the endpoint's rate limit dashboard")
+![Access the dashboard by link in the banner](/img/RLDashboard2.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:47]: Did you want to use the image title as the alt-text?

**Review by vanngo-okta — COMMENTED** [review]
The first string between the square brackets is the alt-text. Did you wanted to use the titles as alt-text?

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 3]
```diff
@@ -67,15 +67,15 @@ The Overview pane provides a summary of the data use for the specified endpoint.
 
 A customer can additionally purchase the rate limit multiplier, which is an add-on feature that allows the customer to multiply an existing rate limit. For example, three times the current rate limit.
 
-![Summary of the rate limit utilization over time pane](/img/RLDashboard4.png "Displays the line graph, bar graph, and timeline graph")
+![Displays the line graph, bar graph, and timeline graph](/img/RLDashboard4.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:70]:
```suggestion
![Displays the line graph, bar graph, and timeline graph of the rate limit utilization over time pane](/img/RLDashboard4.png)
```

**Review by vanngo-okta — APPROVED** [review]
Small suggestion to add context. Otherwise, LGTM!! 🚀

**Review by michaeltamaki-okta — COMMENTED** [review]
Screenshot change looks good to me!

**Review by michaeltamaki-okta — APPROVED** [review]

**Pull request merged**